sormas-ui/src/main/java/de/symeda/sormas/ui/utils/ShowDetailsListener.java

```diff
@@ -49,7 +48,7 @@ public ShowDetailsListener(String detailsColumnId, boolean showOnDoubleClick, Co
 		this.detailsColumnId = detailsColumnId;
 		this.showOnDoubleClick = showOnDoubleClick;
 		this.itemHandler = itemHandler;
-		this.lastHandlingTimestamp = DateTime.now().minusSeconds(1);
+		this.lastHandlingTimestamp = LocalDateTime.now().minusSeconds(1); // make sure the columns are clickable from the beginning
```

Comment:
Don't mix code and comment in one line, put it before. See rule: https://sonarqube.sormas.netzlink.com/coding_rules?languages=java&open=java%3AS139&q=comment